packages/react-components/src/organisms/UserNoEvents.tsx

```diff
@@ -0,0 +1,36 @@
+import { css } from '@emotion/react';
+import { getButtonStyles } from '../button';
+import { crossInCircleIcon } from '../icons';
+import { perRem } from '../pixels';
+
+const titleStyles = css({
+  padding: `20px`,
```

Comment:
Regarding sizing, we use multiples of 3 usually, not sure if you spoke with Manuel and he said to use 20 specifically.